docs/SUMMARY.md

```diff
@@ -55,5 +55,5 @@
   * [Releases](modding-and-development/development/releases.md)
 
 ## Other
-
+* [Helping](helping.md)
```

Comment:
We probably wanna keep the spacer line to be consistent with the rest of the file.

```suggestion

* [Helping](helping.md)
```